porting-postgres.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Porting from PostgreSQL
+summary: Porting an application from PostgreSQL
+toc: false
+---
+
+Although CockroachDB supports PostgreSQL syntax and drivers, it does not offer exact compatibility. This page documents the known list of differences between PostgreSQL and CockroachDB for identical input. That is, a SQL statement of the type listed here will behave differently than in PostgreSQL. Porting an existing application to CockroachDB will require changing these expressions.
+
+Note that some of these differences only apply to rare inputs, and so no change will be needed, even if the listed feature is being used. In these cases, it is safe to ignore the porting instructions.
+
+<div id="toc"></div>
+
+### Overflow of `float`
+
+The `float` type in PostgreSQL will return an error if it overflows or an expression would return Infinity:
+
+```
+postgres=# select 1e300::float * 1e10::float;
+ERROR:  value out of range: overflow
+postgres=#  select pow(0::float, -1::float);
+ERROR:  zero raised to a negative power is undefined
+```
+
+In CockroachDB, these expressions will instead return Infinity:
+
+~~~sql
+SELECT 1e300::float * 1e10::float;
+~~~
+
+```
++----------------------------+
+| 1e300::FLOAT * 1e10::FLOAT |
++----------------------------+
+| +Inf                       |
++----------------------------+
+```
+
+~~~sql
+SELECT pow(0::float, -1::float);
+~~~
+
+```
++---------------------------+
+| pow(0::FLOAT, - 1::FLOAT) |
++---------------------------+
+| +Inf                      |
++---------------------------+
+```
+
+### Precedence of unary `~`
+
+The unary `~` (bitwise not) operator in PostgreSQL has a low precedence. For example, the following query is parsed as `~ (1 + 2)` because `~` has a lower precedence than `+`:
+
+```
+SELECT ~1 + 2
+```
+
+Is parsed as `~ (1 + 2)` because `~` has a lower precedence than `+`.
+
+In CockroachDB, unary `~` has the same (high) precedence as unary `-`, so the above expression will be parsed as `(~1) + 2`.
+
+**Porting instructions:** Manually add parentheses around expressions that depend on the PostgreSQL behavior.
+
+### Precedence of bitwise operators
+
+The operators `|` (bitwise OR), `#` (bitwise XOR), and `&` (bitwise AND) in PostgreSQL all have the same precedence. In CockroachDB the precedence from highest to lowest is: `&`, `#`, `|`.
+
+**Porting instructions:** Manually add parentheses around expressions that depend on the PostgreSQL behavior.
+
+### Integer division
+
+Division of integers in PostgreSQL results in an integer. For example, the following query returns `1`, since the `1 / 2` is truncated to `0` since it is performing integer division:
```

Comment:
I'd actually remove `since it is performing integer division`. Feels redundant and makes the sentence a bit convoluted.